src/components/GameChooser.css

```diff
@@ -36,3 +36,11 @@ input[type="number"]::-webkit-outer-spin-button {
   -moz-appearance: none;
   appearance: none;
 }
+
+.selected-button {
+  font-weight: bold;
+}
+
+.unselected-button {
+  background-color: darkgray;
+}
```

Comment:
instead of a new class this should be added to `.games button`. This should simplify the logic of applying `.selected-button` as well.

Reply:
I was jumping through this hoop so that when no button was selected, they'd all be light grey - when they're dark grey by default they look like they might be disabled. But I just needed to flip it around and only apply a style when there's a selection, but it's not that button. Let me know what you think of this

Reply:
Sounds like you are trying to make a tabbed menu out of it.  In that design case a default would always be selected.  A traditional tabbed menu is maybe not what is needed though.  Touch first is the main design credo and tabs feel more mouse than finger to me.  Also, there are a lot more darts games that could be added so really neither this nor the current design are going to hold up.  Maybe remove this part from your PR for now.